### PR TITLE
Configuring CodeStats

### PR DESCRIPTION
**Pull request by FranDepascuali**
## Summary

**Trello card**: https://trello.com/c/5pYvh9NN/12-configurar-codestats
**Issue**: #24 

Configuring CodeStats for wolmo core.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,3 +6,4 @@ gem 'xcpretty'
 gem 'synx'
 gem 'linterbot'
 gem 'slather'
+gem 'codestats-metrics-reporter', require: nil, git: 'git@github.com:Wolox/codestats-metrics-reporter.git'
```
Comment (epintos) [inline Gemfile]: Here change it to `gem 'codestats-metrics-reporter', require: nil`. Dont forget to run bundle install after

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ## WolMo - Core iOS
 [![Build Status](https://travis-ci.org/Wolox/wolmo-core-ios.svg?branch=master)](https://travis-ci.org/Wolox/wolmo-core-ios)
 [![Coverage Status](https://coveralls.io/repos/github/Wolox/wolmo-core-ios/badge.svg?branch=master)](https://coveralls.io/github/Wolox/wolmo-core-ios?branch=master)
+[![Codestats](http://codestats.wolox.com.ar/organizations/wolox/projects/sonic-words-ios/badge)]((http://codestats.wolox.com.ar/organizations/wolox/projects/sonic-words-ios/badge)
```
Comment (epintos) [inline README.md]: Here replace `sonic-words-ios` to `wolmo-code-ios`

**Comment by woloxci** [comment]
:+1: There are no linter violations.

**Comment by epintos** [comment]
**CodeStats metrics details:**

:x: Code Coverage: 44.09 - Minimum: 70.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/adding-code-stats) for more details.

**Comment by woloxci** [comment]
:+1: There are no linter violations.

**Comment by epintos** [comment]
**CodeStats metrics details:**

:x: Code Coverage: 44.09 - Minimum: 70.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/adding-code-stats) for more details.

**Comment by woloxci** [comment]
:+1: There are no linter violations.

**Comment by epintos** [comment]
**CodeStats metrics details:**

:x: Code Coverage: 44.09 - Minimum: 70.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/adding-code-stats) for more details.

**Comment by woloxci** [comment]
:+1: There are no linter violations.

**Comment by epintos** [comment]
**CodeStats metrics details:**

:x: Code Coverage: 44.09 - Minimum: 70.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/adding-code-stats) for more details.

**Comment by FranDepascuali** [comment]
@epintos I'll lower the coverage limit and add an issue for it.

**Comment by woloxci** [comment]
:+1: There are no linter violations.

**Comment by epintos** [comment]
**CodeStats metrics details:**

:x: Code Coverage: 44.09 - Minimum: 70.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/adding-code-stats) for more details.

**Comment by FranDepascuali** [comment]
@epintos could you take a look on why the coverage requires 70 (I changed it to 40)?

**Comment by epintos** [comment]
@FranDepascuali Put upload_report in false por now and it will work

**Comment by woloxci** [comment]
:+1: There are no linter violations.

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:x: Code Coverage: 44.09 - Minimum: 70.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/adding-code-stats) for more details.

**Comment by woloxci** [comment]
:+1: There are no linter violations.

**Comment by danielaRiesgo** [comment]
Will open a new updated one.

**Pull request closed without merging**